tests/test_taar_locale.py

```diff
@@ -130,3 +181,20 @@ def test_write_output():
     date_filename =\
         "{}{}20171106.json".format(prefix, taar_locale.LOCALE_FILE_NAME)
     assert date_filename in keys
+
+
+@mock_s3
+def test_load_amo_external_whitelist():
+
+    content = [fake_amo_sample_web_extension, fake_amo_sample_legacy]
```

Comment:
This content isn't in the right form, it looks like the database is one large json file that maps GUID to addon object.

Reply:
The full AMO dump is transformed into the lost of guids inside load_amo_external_whitelist().
So I'm stubbing a full amo entry, but the output should be a list of guids.